tests/func/test_ignore.py

```diff
@@ -66,3 +68,49 @@ def test_ignore_in_parent_dir(self):
         all_paths = self._get_all_paths()
 
         self.assertNotIn(forbidden_path, all_paths)
+
+
+def test_metadata_unchanged_when_moving_ignored_file(dvc_repo, repo_dir):
+    new_data_path = repo_dir.DATA_SUB + "_new"
+    with open(
+        os.path.join(dvc_repo.root_dir, DvcIgnore.DVCIGNORE_FILE), "w"
```

Comment:
Just use a separate variable for that path, no need to cram such a giant line in here.